plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/reviews-pagination-numbers/index.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { queryPaginationNumbers as icon } from '@wordpress/icons';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+registerBlockType( metadata, {
```

Comment:
```suggestion
// @ts-expect-error metadata is not typed.
registerBlockType( metadata, {
```

This is missing the usual TS comment for fixing the missing types for `registerBlockType`.